.github/workflows/build.yml

```diff
@@ -227,33 +224,28 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
     - name: Download Build Artifacts
-      uses: dawidd6/action-download-artifact@07ab29fd4a977ae4d2b275087cf67563dfdf0295
+      uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e
       with:
-        run_id: ${{github.run_id}}
         name: Release-${{ matrix.vec.plat }}-windows-2022-x86-${{ matrix.vec.tls }}
         path: artifacts
-        if_no_artifact_found: ignore
     - name: Download Build Artifacts
-      uses: dawidd6/action-download-artifact@07ab29fd4a977ae4d2b275087cf67563dfdf0295
+      uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e
       with:
-        run_id: ${{github.run_id}}
         name: Release-${{ matrix.vec.plat }}-windows-2022-x64-${{ matrix.vec.tls }}
         path: artifacts
-        if_no_artifact_found: ignore
     - name: Download Build Artifacts
-      uses: dawidd6/action-download-artifact@07ab29fd4a977ae4d2b275087cf67563dfdf0295
+      uses: actions/download-artifact@95815c38cf2ff2164869cbab79da8d1f422bc89e
+      if: matrix.vec.tls != 'quictls' # TODO: Remove once quictls is supported
```

Comment:
Are these TODOs for this PR or a future one?

Reply:
A future PR. I cleaned up how these things worked. If you look above, you will see build for arm64 is also a TODO. I hope that by moving to the latest version of quictls/openssl we can remove these.